### ISSUE TITLE
mkdocs-bootswatch==0.4.0 is not compatible with mkdocs==0.17.0

### ISSUE THREAD

**Issue by sergeykhegay** [mkdocs/mkdocs-bootswatch#35]
Here what I get with the above versions. Seems to work fine with the `mkdocs==0.16.3`.

```
skhegay@C02T52AWFVH4-lm (fix-dependency)✗ % mkdocs serve                                                                       ~/Projects/yamas_userguide_2.0
INFO    -  Building documentation...
WARNING -  The theme 'yeti' does not appear to have a configuration file. Please upgrade to a current version of the theme.
INFO    -  Cleaning site directory
INFO    -  Template skipped: 'search.html'. Not found in template directories.
ERROR   -  Error building page index.md
Traceback (most recent call last):
  File "/usr/local/bin/mkdocs", line 11, in <module>
    sys.exit(cli())
  File "/Library/Python/2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Library/Python/2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Python/2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Python/2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/mkdocs/__main__.py", line 127, in serve_command
    livereload=livereload
  File "/Library/Python/2.7/site-packages/mkdocs/commands/serve.py", line 114, in serve
    config = builder()
  File "/Library/Python/2.7/site-packages/mkdocs/commands/serve.py", line 109, in builder
    build(config, live_server=live_server, dirty=dirty)
  File "/Library/Python/2.7/site-packages/mkdocs/commands/build.py", line 282, in build
    build_pages(config, dirty=dirty)
  File "/Library/Python/2.7/site-packages/mkdocs/commands/build.py", line 242, in build_pages
    _build_page(page, config, site_navigation, env)
  File "/Library/Python/2.7/site-packages/mkdocs/commands/build.py", line 150, in _build_page
    template = env.get_template('main.html')
  File "/Library/Python/2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Library/Python/2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Library/Python/2.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Library/Python/2.7/site-packages/jinja2/loaders.py", line 187, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: main.html
```

**Comment by lucasdp** [comment]
`WARNING -  Your theme does not appear to contain a 'main.html' template. The 'base.html' template was used instead, which is deprecated. Update your theme so that the primary entry point is 'main.html'.
WARNING -  Template variable warning: 'google_analytics' is being deprecated and will not be available in a future version. Use 'config.google_analytics' instead.
WARNING -  Template variable warning: 'copyright' is being deprecated and will not be available in a future version. Use 'config.copyright' instead.
WARNING -  Template variable warning: 'favicon' is being deprecated and will not be available in a future version. Use '{{ base_url }}/img/favicon.ico' instead.
WARNING -  Template variable warning: 'site_name' is being deprecated and will not be available in a future version. Use 'config.site_name' instead.
WARNING -  Template variable warning: 'page_description' is being deprecated and will not be available in a future version. Use 'config.site_description' instead.
WARNING -  Template variable warning: 'canonical_url' is being deprecated and will not be available in a future version. Use 'page.canonical_url' instead.
WARNING -  Template variable warning: 'page_title' is being deprecated and will not be available in a future version. Use 'page.title' instead.
WARNING -  Template variable warning: 'site_author' is being deprecated and will not be available in a future version. Use 'config.site_author' instead.
WARNING -  Template variable warning: 'next_page' is being deprecated and will not be available in a future version. Use 'page.next_page' instead.
WARNING -  Template variable warning: 'repo_url' is being deprecated and will not be available in a future version. Use 'config.repo_url' instead.
WARNING -  Template variable warning: 'previous_page' is being deprecated and will not be available in a future version. Use 'page.previous_page' instead.
WARNING -  Template variable warning: 'repo_name' is being deprecated and will not be available in a future version. Use 'config.repo_name' instead.
WARNING -  Template variable warning: 'homepage_url' is being deprecated and will not be available in a future version. Use 'nav.homepage.url' instead.
WARNING -  Template variable warning: 'toc' is being deprecated and will not be available in a future version. Use 'page.toc' instead.
WARNING -  Template variable warning: 'content' is being deprecated and will not be available in a future version. Use 'page.content' instead.
WARNING -  Template variable warning: 'meta' is being deprecated and will not be available in a future version. Use 'page.meta' instead.`

Those are the warnings in MkDocs==0.16

**Comment by sergeykhegay** [comment]
@lucasdp, apparently, they enforced those warnings in the major release.

**Comment by abhinvsinh** [comment]
@lucasdp, @sergeykhegay  : Even with the warnings the project used to get compiled in the previous version, with the new version it fails.
I am getting the same error with the theme "cosmo".

 doc_build runtests: commands[0] | mkdocs build -d artifacts/site
WARNING -  The theme 'cosmo' does not appear to have a configuration file. Please upgrade to a current version of the theme. 
 INFO    -  Cleaning site directory 
 INFO    -  Building documentation to directory: /var/builds/workspace/235250-v3-PR-13/BUILD_CONTAINER/python/label/DOCKER-LOW/app_root/builds/artifacts/site 
 INFO    -  Template skipped: 'search.html'. Not found in template directories. 
 ERROR   -  Error building page index.md

**Comment by Tslimshady99** [comment]
Any Fix?

**Comment by ketozhang** [comment]
What's the update on this. So far the new MkDocs update has been killing every theme out there except Materials and probably other few.

**Comment by robodude666** [comment]
@ketozhang its been a while, but it looks like the `master` branch is updated. You can manually clone the repo or download the zip. It looks like the latest version wasn't pushed to `pypi` yet.

**Comment by Serhioromano** [comment]
I have the same issue. Cannot use bootswatch themes. Same main.html problem.

**Comment by waylan** [comment]
I just released version 0.5 which resolves this issue. Enjoy!
